src/js/components/AccordionPanel/doc.js

```diff
@@ -31,6 +31,11 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: '<FormDown />',
   },
+  'accordion.icons.color': {
```

Comment:
Nice catch buddy, thanks! 
Can we move this object to line 29? so it will be compliant with alphabetical order?